backend/package.json

```diff
@@ -12,12 +12,17 @@
     "start:dev": "nest start --watch",
```

Comment:
there is no `lint-staged` lage in backend :eye: 

https://github.com/framgia/sph-els-mark/blob/929945e0b5a01da254a333d83d9cdc325ebe96c3/frontend/package.json#L64-L67